.github/workflows/lint-js-css.yml

```diff
@@ -5,6 +5,11 @@ on:
     branches: [main]
   pull_request:
     branches: [main]
+    paths:
+      - '**.css'
+      - '**.js'
+      - '**.json'
+      - '**.yml'
```

Comment:
`json` and `yml` is added to detect changes in workflows, package updates and block updates